optimade/models/structures.py

```diff
@@ -39,8 +39,8 @@
 EPS = 2**-23
 
 
-Vector3D = conlist(float, min_items=3, max_items=3)
-Vector3D_unknown = conlist(Union[float, None], min_items=3, max_items=3)
+Vector3D = conlist(float, min_items=3, max_items=3)  # type: ignore[valid-type]
+Vector3D_unknown = conlist(Union[float, None], min_items=3, max_items=3)  # type: ignore[valid-type]
```

Comment:
```suggestion
Vector3D = conlist(float, min_items=3, max_items=3)
Vector3D_unknown = conlist(Union[float, None], min_items=3, max_items=3)
```
On my machine it is not necessary to add type ignore here.